<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleshipGUI</class>
 <widget class="QMainWindow" name="BattleshipGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instructions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="shipGroup">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(46, 204, 113, 0.4);
</string>
    </property>
    <property name="title">
     <string>SHIPS</string>
    </property>
    <widget class="QPushButton" name="ship1">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left;</string>
     </property>
     <property name="text">
      <string>ship1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>100</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left;</string>
     </property>
     <property name="text">
      <string>ship4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>75</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left;</string>
     </property>
     <property name="text">
      <string>ship3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>150</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>125</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship8">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>200</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>175</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship10">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>250</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ship9">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>225</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(213, 50, 92, 0.8);
Text-align:left</string>
     </property>
     <property name="text">
      <string>ship9</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="player1winlose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WIN</string>
    </property>
   </widget>
   <widget class="QTableView" name="player1board">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>222</width>
      <height>222</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotDotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="player2label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commonButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player1label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startgame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(46, 204, 113, 0.4);
</string>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2winlose">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Marker Felt</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>440</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>DONE</string>
    </property>
   </widget>
   <widget class="QTableView" name="player2board">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>222</width>
      <height>222</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotDotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WELCOME TO BATTLESHIP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SHOW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>410</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(194, 225, 107,1)</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QTableView" name="player2shotboard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>222</width>
      <height>222</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotDotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="player1shotboard">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>222</width>
      <height>222</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashDotDotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
